client/forms.py

```diff
@@ -54,23 +54,29 @@ class VolunteerForm(forms.Form):
     )
 
     my_validator = RegexValidator(r"^\d+$")
-    full_name = forms.CharField(max_length=200)
-    identity = forms.CharField(max_length=9, validators=[my_validator])
+    #full_name = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
+    first_name = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
+    last_name = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
+    identity = forms.CharField(max_length=ID_LENGTH, validators=[my_validator])
+    organization = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH, required=False)
     age = forms.IntegerField()
-    area = forms.MultipleChoiceField(choices = AREAS, widget=forms.CheckboxSelectMultiple())
+    area = forms.MultipleChoiceField(choices=AREAS, widget=forms.CheckboxSelectMultiple())
     languages = forms.MultipleChoiceField(choices = get_the_lang_choices, widget=forms.CheckboxSelectMultiple())
-    phone_number = forms.CharField(max_length=200)
-    email = forms.CharField(max_length=200)
-    city = forms.ChoiceField(choices = CITIES)
-    address = forms.CharField(max_length=200)
+    phone_number = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
+    email = forms.CharField(max_length=DEFAULT_MAX_FIELD_LENGTH)
```

Comment:
Consider changing email to EmailField

Reply:
See above - I didn't really want to change any of the field names and so on. Maybe we can open a seperate issue that's about cleaning things up a bit, there's plenty to go around